components/msg/constellation_msg.rs

```diff
@@ -162,6 +163,15 @@ pub enum TraversalDirection {
     Back(usize),
 }
 
+pub struct HistoryEntry {
+    pub url: ServoUrl
```

Comment:
Seems sensible. It might be nice to reuse some of the existing structures rather than repeat them here.